### ISSUE TITLE
Have someone installed onto a Cpanel environment?

### ISSUE THREAD

**Issue by doug3d** [taigaio/taiga-back#1182]
Hi guys, i want install Taiga onto one of my servers. Today im preparing one totally empty, but in few weeks i will try to do on my Cpanel (not the main one) server.

Anyone had tried ? It worked?

**Comment by TheBlapse** [comment]
Hey @doug3d, Did it work with cpanel?

**Comment by doug3d** [comment]
@TheBlapse   hey man. sorry for the late answer.  As i remember i succeded. but i changed my servers  from cpanel to plesk.

**Comment by MatDepInfo** [comment]
So @TheBlapse, is it working with plesk ? How do you achieve it ?

**Comment by TheBlapse** [comment]
@MatDepInfo 
I don't use plesk

**Comment by MatDepInfo** [comment]
@TheBlapse I can use both of them but I though it was you but in fact it is @doug3d who use Plesk.

**Comment by doug3d** [comment]
Yeah, i changed my env  to plesk last year.  And i give up to taigaio.  but
i cant see if work on plesk on next week

Em ter., 11 de jan. de 2022 às 08:06, MatDepInfo ***@***.***>
escreveu:
